### PR TITLE
feature: support ENTER compiler directive.

### PR DESCRIPTION
**Pull request by ap891843**
Signed-off-by: ap891843 <aman.prashant@broadcom.com>

### CODE REVIEW

File: server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolLexer.g4 [hunk 1]
```diff
@@ -54,8 +55,8 @@ channels{TECHNICAL}
     {
       String input = _input
                      .getText(Interval.of(_tokenStartCharIndex, _input.index()))
-                     .replace("\r","").replace("\n","").trim();
-      if(StringUtils.isBlank(input.substring(5, input.length()-1)))
+                     .replace("\r","").replace("\n","").replace(".","").trim();
```
Comment (temanbrcom) [inline server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolLexer.g4:58]: Try using [`\R`](https://stackoverflow.com/questions/11644100/regular-expression-does-not-match-newline-obtained-from-formatter-object) here.

File: server/src/test/java/org/eclipse/lsp/cobol/usecases/TestEnterCompilerDirective.java [hunk 2]
```diff
@@ -50,9 +55,14 @@ void testErrorOnEnterCompilerDirective_whenLanguage() {
         ImmutableMap.of(
             "1",
             new Diagnostic(
-                new Range(new Position(1, 7), new Position(1, 14)),
+                new Range(new Position(1, 7), new Position(1, 23)),
```
Comment (temanbrcom) [inline server/src/test/java/org/eclipse/lsp/cobol/usecases/TestEnterCompilerDirective.java:58]: Why does it point to a position that is further than the line end?

Comment (ap891843) [reply server/src/test/java/org/eclipse/lsp/cobol/usecases/TestEnterCompilerDirective.java]: Corrected. 
Problem:
DOT_FS was consuming till the next line. So for below code
`       Identification Division.
       ENTER .
       Program-Id. control-dir.`

mapping token had below token for enter stmt.
`ENTER .
       `

File: server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolLexer.g4 (hunk 1 above)
Comment (ap891843) [reply server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolLexer.g4]: Done.

**Comment by ap891843** [comment]
> Please, change the prefix to `feat:` and remove the dot at the end to match the commit message agreement.

Done

**Review by temanbrcom — APPROVED** [review]
🌊

**Review by grianbrcom — APPROVED** [review]
:raccoon:

File: server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolLexer.g4 (hunk 1 above)
Comment (grianbrcom) [reply server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolLexer.g4]: Wow! Didn't know about this feature.

**Pull request merged**